src/components/Editor/EditorInput/EditorInput.js

```diff
@@ -82,11 +82,11 @@ const EditorInput = () => {
                     panelIndex={panelIndex}
                   >
                     <span
-                      className={`react-tabs__tab-inner${fileName !== 'main.py'? ' react-tabs__tab-inner--split': ''}`}
+                      className={`react-tabs__tab-inner${!["main.py", "index.html"].includes(fileName) ? ' react-tabs__tab-inner--split': ''}`}
```

Comment:
won't be on the main branch yet (it's in the mvp branch) but I added a package called `classnames`, that allows us to do something like this now:

```
classNames(
    'react-tabs__tab-inner',
    {'react-tabs__tab-inner--split': !["main.py", "index.html"].includes(fileName)}
);
```

which I think is tidier in the long run, especially if you end up with multiple optional classes

Reply:
Yes!! Completely forgot you added that! Love `classNames` it makes things like this way nicer to read. Cheers